### DISCUSSION TITLE
How does mydumper deal with GTID?

### DISCUSSION THREAD

**Discussion by astesin** [mydumper/mydumper#1320] [General]
Yes, I know mydumper correctly processes the binlog name and position, this is cool and extremely useful for setting up replicas. Thanks! :)

But what about a GTID? I.e. if I am backing up a server, does it store the server-provided GTID at the moment of FTWRL lock? So that when setting up a replica, I can use it instead of binlog name + pos.

Thanks in advance! Regards, Andreas

**Comment by tanji** [comment]
GTID gets written into the `metadata` file, as long as your backup is consistent you can use it for replication purposes.
